README.md

```diff
@@ -1,6 +1,7 @@
 # Hacktober Fest 
```

Comment:
Don't edit the readme, remove this commit from your PR

Reply:
Okay